docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3'
+services:
+  db:
+    image: mdillon/postgis:10-alpine
+    container_name: postgres01
+    ports:
+      - '5432'
```

Comment:
Can we also publish port for postgis?

Reply:
PostGIS doesn't require any other ports to be open, unless I misunderstood your question?